src/components/map/map-control.js

```diff
@@ -305,18 +319,21 @@ const MapControlFactory = () => {
       onToggleSplitMap: PropTypes.func.isRequired,
       onToggleMapControl: PropTypes.func.isRequired,
       onSetEditorMode: PropTypes.func.isRequired,
+      onToggleEditorVisibility: PropTypes.func.isRequired,
       top: PropTypes.number.isRequired,
 
       // optional
       readOnly: PropTypes.bool,
       scale: PropTypes.number,
       mapLayers: PropTypes.object,
-      editor: PropTypes.object
+      editor: PropTypes.object,
+      indexMap: PropTypes.number
```

Comment:
`indexMap` is defined, but `index` is passed in

Reply:
actually I noticed we were already passing mapIndex hence we don't need either index now indexMap

Reply:
delete `indexMap: PropTypes.number`